core/src/main/scala/org/apache/spark/storage/BlockObjectWriter.scala

```diff
@@ -117,7 +117,7 @@ private[spark] class DiskBlockObjectWriter(
 
   /** Calling channel.position() to update the write metrics can be a little bit expensive, so we
```

Comment:
This comment is a bit out of place now. Maybe put this down where the actual "% 32" check occurs.